versions.tf

```diff
@@ -19,31 +19,23 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.66, < 5.22"
+      version = "~> 5.8"
```

Comment:
Should mark this as a breaking change: `chore!:`.

I'd also suggest using ">= 5.8, < 6" so a automated PR can be opened to bump 6>7 when TPG v6 is released.

Reply:
Good idea, added constraint.